### PR TITLE
Remove credentials setting and update docs

### PR DESCRIPTION
**Pull request by ildac**
related to #302 

Changes in this pull request:
- remove credentials line from build.sbt
- update "Sonatype Credentials" section in MAINTANERS.md documentation

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -253,8 +253,7 @@ lazy val corePublishSettings = Seq(
     Developer(id="mitruly", name="Michael Ly", email="michaeltrulyng@gmail.com", url=url("https://github.com/mitruly")),
     Developer(id="britneywright", name="Britney Wright", email="blw06g@gmail.com", url=url("https://github.com/britneywright")),
   ),
-  sonatypeProfileName := "io.vinyldns",
-  credentials += Credentials(Path.userHome / ".sbt" / "1.0" / "vinyldns-gpg-credentials")
+  sonatypeProfileName := "io.vinyldns"
```
Comment (pauljamescleary) [inline build.sbt]: should we also remove the `.dockerCredentials`?

**Comment by pauljamescleary** [comment]
👋  @ildac , thanks for helping us out.  Left a question, also maybe @nimaeskandary can answer as well.

File: build.sbt (hunk 1 above)
Comment (ildac) [reply build.sbt]: Yes I think that could use the same global setup as the Sonatype credential, I can give it a trt and if the case update the pull request.

Comment (pauljamescleary) [reply build.sbt]: Thanks for looking into this.  I _think_ ideally we don't want _any_ credentials in the build, and just assume they live in global

Comment (nimaeskandary) [reply build.sbt]: This global route makes sense to me

**Pull request merged**